ports/zephyr/mpconfigport.h

```diff
@@ -68,6 +68,8 @@
 #define MICROPY_PY_MACHINE_PIN_MAKE_NEW mp_pin_make_new
 #define MICROPY_PY_MACHINE_UART     (1)
 #define MICROPY_PY_MACHINE_UART_INCLUDEFILE "ports/zephyr/machine_uart.c"
+#define MICROPY_PY_MACHINE_WDT      (1)
```

Comment:
Maybe this could be wrapped in `#if CONFIG_WATCHDOG` ... `#endif` so that it's only enabled if the watchdog is enabled in the Kconfig?

Reply:
I agree. Thanks!